src/components/DatePicker/components/Month/tests/Month.test.tsx

```diff
@@ -11,7 +10,7 @@ describe('<Month />', () => {
   describe('title', () => {
     it('passes the abbreviated value to the title prop of Weekday', () => {
       const month = mountWithAppProvider(
-        <Month month={0} year={2018} weekStartsOn={Weekdays.Monday} />,
+        <Month month={0} year={2018} weekStartsOn={1} />,
```

Comment:
So this is where we're avoiding casting now? I still prefer the enum as a consumer I think. It's no super clear that 1 represents Monday

Reply:
Turns out I was a little wrong with my original assertions.

Doing it either of these ways was always possible in consumers (and continues to be so for the moment). However we never exported the `Weekdays` enum, consumers had to get at it via `import {Weekdays} from '@shopify/javascript-utilities/date';`. which was practically never done (`Weekday.SOMEDAY` was used 3 times in web, two of which are are in the tests of a forked DatePicker).

This change didn't need to occur, but it means I can avoid import `Weekdays` within this test file